chaser:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Multi Controllers ---------------------------------------
  joint1_multi_controller:
    type: velocity_controllers/JointMultiController
    # type: velocity_controllers/JointVelocityController
    # type: effort_controllers/JointMultiController                        # default
    joint: arm_joint_1
    # pid: {p: 0.01, i: 5.0, d: 30.0}     # For JointPositionController    # default
    # pid_v: {p: 10.0, i: 10.0, d: 0.0}   # For JointVelocityController

  joint2_multi_controller:
    type: velocity_controllers/JointMultiController
    # type: velocity_controllers/JointVelocityController
    # type: effort_controllers/JointMultiController                        # default
    joint: arm_joint_2
    # pid: {p: 30.0, i: 10.0, d: 60.0}    # For JointPositionController    # default
    # pid_v: {p: 10.0, i: 10.0, d: 0.0}   # For JointVelocityController    # default

  joint3_multi_controller:
    type: velocity_controllers/JointMultiController
    # type: velocity_controllers/JointVelocityController
    # type: effort_controllers/JointMultiController                        # default
    joint: arm_joint_3
    # pid: {p: 30.0, i: 10.0, d: 60.0}    # For JointPositionController    # default
    # pid_v: {p: 10.0, i: 10.0, d: 0.0}   # For JointVelocityController    # default